web/src/beta/features/PluginPlayground/Plugins/index.tsx

```
@@ -34,99 +34,139 @@ const Plugins: FC<Props> = ({
+      </IconList>
+      <PluginListWrapper>
+        <PluginList>
+          {customPlugin && (
+            <div>
+              {customPlugin.plugins.map((plugin) => (
+                <PluginEntryItem
+                  plugin={plugin}
+                  selectedPluginId={selectedPlugin.id}
+                  onSelect={selectPlugin}
+                  onShare={handleShareIconClicked}
+                />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)